content/post/22-04-30-progress-report-33.md

```diff
@@ -0,0 +1,287 @@
++++
+title = "Progress Report #33: Delta Variant"
+author = "DrSmugleaf, ike709, and Timrod"
+date = 2022-04-30
```

Comment:
necessary change to display it in the preview (hugo will hide it if the releasedate hasnt happened yet)
```suggestion
date = 2022-04-29
```